torchci/components/JobLinks.tsx

```diff
@@ -301,3 +314,22 @@ function DisableIssue({
     </a>
   );
 }
+
+function RevertInfoCopy({ job }: { job: JobData }) {
+  const info = [];
+  const testName = getTestName(
+    (job.failureLines && job.failureLines[0]) ?? "",
+    false,
+    true
+  );
+  if (testName !== null) {
+    info.push(testName);
+  }
+  info.push(job.htmlUrl);
+  info.push(`https://hud.pytorch.org/${job.repo}/commit/${job.sha}`);
+  return CopyLink({
```

Comment:
What do you think about making this button conditional on the job actually failing? Right now it shows up even on pending jobs, which feels rather noisey